### PR TITLE
Branch calendar

### PR DESCRIPTION
**Pull request by harithadiv**
- Command eg. : calendar s/1 ay/19-20
- lists out all the events that falls under the semester and academic year 
- still working on printing out in a calendar format, only lists all the events now.

### CODE REVIEW

File: Duke.log.4 [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="windows-1252" standalone="no"?>
```
Comment (benchan911) [inline Duke.log.4]: Should remove your log file

File: src/main/java/seedu/calendar/AddSecondSemester.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package seedu.calendar;
+
+import seedu.command.Command;
+import seedu.event.EventList;
+import seedu.exception.DukeException;
+
+public class AddSecondSemester extends Command {
```
Comment (benchan911) [inline src/main/java/seedu/calendar/AddSecondSemester.java:7]: What is the difference between addfirstsemester and addsecondsemester? Do the prof need the distinction between them?

Comment (benchan911) [reply src/main/java/seedu/calendar/AddSecondSemester.java:7]: Or can it be a year long thing?

**Review by benchan911 — CHANGES_REQUESTED** [review]
Please remove log file first before merging, I think its fine

File: src/main/java/seedu/parser/CalendarParser.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+public class CalendarParser {
+    private static int semester;
+
+    public CalendarParser() {
+    }
+
+    public static String[] parseDescription(String parameters) throws DukeException {
+        String[] tokens = parameters.split(" ");
+        if (tokens.length > 2 || tokens.length < 2) {
```
Comment (lowjiayee) [inline src/main/java/seedu/parser/CalendarParser.java]: Will (tokens.length != 2) summarise this condition?

**Pull request merged**